relay-redis/src/real.rs

```diff
@@ -382,3 +392,160 @@ pub struct Stats {
     /// The number of idle connections.
     pub idle_connections: u32,
 }
+
+/// [`bb8::ManageConnection`] for an async redis server running in single mode.
+/// Takes a [`AsyncConnectionConfig`] which is applied to every connection obtained through
+/// the pool.
+pub struct RedisConnectionManagerWithConfig {
+    client: Client,
+    config: AsyncConnectionConfig,
+}
+
+impl RedisConnectionManagerWithConfig {
+    /// Creates a new [`RedisConnectionManagerWithConfig`] for the given client and config.
+    pub fn new(client: Client, config: AsyncConnectionConfig) -> Self {
+        RedisConnectionManagerWithConfig { client, config }
+    }
+}
+
+#[axum::async_trait]
```

Comment:
Let's add the `async-trait` crate directly and not use a re-export, also means the Redis crate does not depend on a web framework.